templates/tour.html

```diff
@@ -234,7 +234,7 @@
                 <p>MAAS can easily manage the most complex network environments, including all your:</p>
                 <div class="two-col">
                     <ul>
-                        <li><abbr title="Internet Protocol version 4">IPv4</abbr> and <a href="https://docs.ubuntu.com/maas/2.1/en/installconfig-network-ipv6" class="external" aria-label="External link to install and configure network Internet Protocol version 6 on MAAS documentation"><abbr title="Internet Protocol version 6">IPv6</abbr></a></li>
+                        <li><abbr title="Internet Protocol version 4">IPv4</abbr> and <a href="https://docs.ubuntu.com/maas/2.2/en/installconfig-network-ipv6" class="external" aria-label="External link to install and configure network Internet Protocol version 6 on MAAS documentation"><abbr title="Internet Protocol version 6">IPv6</abbr></a></li>
                         <li><a href="https://docs.ubuntu.com/maas/devel/en/installconfig-network-subnet-management" class="external" aria-label="External link to install and configure network subnet management on MAAS documentation">Subnets</a></li>
                         <li><a href="https://docs.ubuntu.com/maas/devel/en/intro-concepts#vlans" class="external" aria-label="External link to intro concepts about virtual local area networks on MAAS documentation"><abbr title="Virtual local area network">VLAN</abbr></a></li>
```

Comment:
@nottrobin I'll bring it up in stand up with the MAAS team as I believe this should always be pointing to devel.